topdown/array.go

```diff
@@ -71,7 +71,25 @@ func builtinArraySlice(a, i, j ast.Value) (ast.Value, error) {
 	return arr.Slice(startIndex, stopIndex), nil
 }
 
+func builtinArrayReverse(bctx BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	arr, err := builtins.ArrayOperand(operands[0].Value, 1)
+	if err != nil {
+		return err
+	}
+
+	length := arr.Len()
+	reversedArr := make([]*ast.Term, length)
+
+	for index := 0; index < length; index++ {
+		reversedArr[index] = arr.Elem(length - index - 1)
+	}
+
+	return iter(ast.ArrayTerm(reversedArr...))
+
```

Comment:
Can we skip the extra newline after the return here and in `builtinReverse` below?